### PR TITLE
Add View Feature

### LINKED ISSUE #2: As a user I can view all details of my selected Applicant
... so that I don’t need to open and work with multiple files

### PR DESCRIPTION
**Pull request by maryjess**
- Update UI to show viewed person window
- Add `ViewCommand.java`
- Add `ViewCommandParser.java`
- Update UI picture in User Guide and README.md
- Add JUnit tests for `ViewCommand` and `ViewCommandParser`

Resolves #2

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Gradle build files
 /.gradle/
 /build/
+/bin/
```
Comment (cliftonfelix) [inline .gitignore:4]: bin folder is still inside. Please fix it.

Comment (maryjess) [reply .gitignore:4]: Yup removed!

**Review by lulucopter — CHANGES_REQUESTED** [review]
Overall, LGTM, but noticed that the codecov  CI/CD test failed(?)

Suggested some possible test cases you can add in as well to your Parser class to increase coverage.

As for UI test coverage, not sure if its part of the tP project scope (I think prof did it in AB4). Not sure what to do in this case to pass the codecov tests

File: src/main/java/seedu/address/ui/PersonCard.java [hunk 2]
```diff
@@ -62,14 +62,14 @@ public PersonCard(Person person, int displayedIndex) {
...
-        cap.setText(person.getCap().toString());
-        address.setText(person.getAddress().value);
-        university.setText(person.getUniversity().value);
-        major.setText(person.getMajor().value);
+        // gender.setText(person.getGender().value);
+        // graduationDate.setText(person.getGraduationDate().value);
+        // cap.setText(person.getCap().toString());
+        // address.setText(person.getAddress().value);
+        // university.setText(person.getUniversity().value);
+        // major.setText(person.getMajor().value);
```
[the remark this answers is not in the document]
Comment (maryjess) [reply src/main/java/seedu/address/ui/PersonCard.java:72]: Yup, will remove dead code once the UI is finished

**Review by cliftonfelix — APPROVED** [review]
LGTM. Well done!

**Pull request merged**